nn/conformer.py

```diff
@@ -14,21 +14,20 @@ class _PositionwiseFeedForward(nn.Module):
       FF -> Activation -> Dropout -> FF
   """
 
-  def __init__(self, dim_model: int, dim_ff: int, dropout: float, activation, l2: float = 0.0):
+  def __init__(self, dim_model: int, dim_ff: int, dropout: float, activation):
```

Comment:
See my comments on Transformer on the naming of `dim_model`. I think `out_dim` is maybe nicer.

Also, once we have #17, the type (`int`) would change here.